src/components/PostList/PostList.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Post } from '../Post';
+
+export const PostList = ({ posts }) => (
+  <div className="posts">
+    {posts.map(post => <div key={post.id}><Post {...post} /></div>)}
```

Comment:
```suggestion
    {posts.map(post => <Post key={post.id} {...post} />)}
```
do you need this `div`?